packages/models-library/src/models_library/progress_bar.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any, ClassVar, Literal, TypeAlias
+
+from pydantic import BaseModel, Extra
+
+# NOTE: keep a list of possible unit, and please use correct names
+ProgressUnit: TypeAlias = Literal["Byte"]
+
+
+class ProgressReport(BaseModel):
+    actual_value: float
+    total: float
+    unit: ProgressUnit | None = None
+
+    @property
+    def percent_value(self) -> float:
+        if self.total != 0:
+            return max(min(self.actual_value / self.total, 1.0), 0.0)
+        return 0
+
+    class Config:
+        frozen = True
+        extra = Extra.forbid
```

Comment:
why so restrictive on parsing (i.e. `extra=Extra.forbid` ) ? it can be handy to ignore input values in some situations

Reply:
removed